diff --git a/node_modules/react-native-kline-view/index.d.ts b/node_modules/react-native-kline-view/index.d.ts
index 549252a..9f4cd53 100644
--- a/node_modules/react-native-kline-view/index.d.ts
+++ b/node_modules/react-native-kline-view/index.d.ts
@@ -1,3 +1,4 @@
+import type React from "react";
 import type {
   HostComponent,
   NativeSyntheticEvent,
@@ -172,6 +173,6 @@ export interface RNKLineViewProps extends ViewProps {
 
 export type RNKLineViewComponent = HostComponent<RNKLineViewProps>;
 
-declare const RNKLineView: (props: RNKLineViewProps) => ReactNode;
+declare const RNKLineView: React.ForwardRefExoticComponent<RNKLineViewProps & React.RefAttributes<React.ComponentRef<RNKLineViewComponent>>>;
 
 export default RNKLineView;
diff --git a/node_modules/react-native-kline-view/index.js b/node_modules/react-native-kline-view/index.js
index 0310165..bbf535f 100644
--- a/node_modules/react-native-kline-view/index.js
+++ b/node_modules/react-native-kline-view/index.js
@@ -3,7 +3,7 @@ import { View, StyleSheet, requireNativeComponent } from "react-native";
 
 const RNKLineViewNative = requireNativeComponent("RNKLineView");
 
-function RNKLineView({ tradeComponent, style, ...props }) {
+const RNKLineView = React.forwardRef(function RNKLineView({ tradeComponent, style, ...props }, ref) {
   const [markers, setMarkers] = useState([]);
 
   const onTradeMarkersLayout = useCallback((event) => {
@@ -13,6 +13,7 @@ function RNKLineView({ tradeComponent, style, ...props }) {
   return (
     <View style={style}>
       <RNKLineViewNative
+        ref={ref}
         {...props}
         style={StyleSheet.absoluteFill}
         useCustomTradeMarker={!!tradeComponent}
@@ -42,6 +43,6 @@ function RNKLineView({ tradeComponent, style, ...props }) {
         })}
     </View>
   );
-}
+});
 
 export default RNKLineView;
