diff --git a/node_modules/native-kline-view/android/src/main/java/com/github/fujianlian/klinechart/HTKLineConfigManager.java b/node_modules/native-kline-view/android/src/main/java/com/github/fujianlian/klinechart/HTKLineConfigManager.java
index 6e7dfa8..a3be280 100644
--- a/node_modules/native-kline-view/android/src/main/java/com/github/fujianlian/klinechart/HTKLineConfigManager.java
+++ b/node_modules/native-kline-view/android/src/main/java/com/github/fujianlian/klinechart/HTKLineConfigManager.java
@@ -62,7 +62,7 @@ public class HTKLineConfigManager {
 
     public boolean useCustomTradeMarker = false;
 
-    public Callback onTradeMarkersLayout;
+    public HTKLineCallback onTradeMarkersLayout;
 
 
 
diff --git a/node_modules/native-kline-view/android/src/main/java/com/github/fujianlian/klinechart/container/HTKLineContainerView.java b/node_modules/native-kline-view/android/src/main/java/com/github/fujianlian/klinechart/container/HTKLineContainerView.java
index 737bd3f..e7f4813 100644
--- a/node_modules/native-kline-view/android/src/main/java/com/github/fujianlian/klinechart/container/HTKLineContainerView.java
+++ b/node_modules/native-kline-view/android/src/main/java/com/github/fujianlian/klinechart/container/HTKLineContainerView.java
@@ -159,7 +159,7 @@ public class HTKLineContainerView extends RelativeLayout {
                 );
             }
         };
-        configManager.onTradeMarkersLayout = new Callback() {
+        configManager.onTradeMarkersLayout = new HTKLineCallback() {
             @Override
             public void invoke(Object... args) {
                 @SuppressWarnings("unchecked")
